### ISSUE TITLE
Enhance Attribution in MD_DataIdentification

### ISSUE THREAD

**Issue by mwengren** [ioos/catalog-ckan#145]
We've had requests from RAs that the NDBC SOS records in the Catalog reflect attribution of the RA as the data 'originator' or original publisher, or whatever the appropriate term should be.  

The sensorml2sio module has been updated to include that info at the following xpath:
```
//gmd:MD_DataIdentification/gmd:citation/gmd:CI_Citation/gmd:citedResponsibleParty/gmd:CI_ResponsibleParty/gmd:role/gmd:CI_RoleCode(@codeListValue="originator")
```

Can we look into what it would take to parse this from the XML (as well as any other gmd:citedResponsibleParty elements with similar xpath under MD_DataIdentification with other codeListValues - 'publisher' in this case would be NDBC) and display somewhere in the UI?  Probably under the 'Access' tab, which currently includes a 'Distributor' credit.  

Here's an example record: 

https://data.ioos.us/dataset/waimea-bay-hi-106

Registry XML: https://registry.ioos.us/waf/NOAA%20NDBC/sdf.ndbc.noaa.gov-urn_ioos_station_wmo_51201.xml

**Comment by lukecampbell** [comment]
I think it's certainly doable. @benjwadams could take a look. 

It's all defined in [ckanext-spatial's model](https://github.com/ioos/ckanext-spatial/blob/master/ckanext/spatial/model/harvested_metadata.py) and [the harvester](https://github.com/ioos/catalog-ckan/blob/master/ckanext/ioos_theme/harvesters/ioos_harvester.py) to pass that information to the UI. 

From there the jinja templates pull it out and make use of it like [this one](https://github.com/ioos/catalog-ckan/blob/master/ckanext/ioos_theme/templates/package/snippets/meta_access.html) for the access tab.

**Comment by lukecampbell** [comment]
Just a heads up this is ready to go, I can deploy tonight if everyone's ok with that?

@mwengren @rsignell-usgs @benjwadams

**Comment by mwengren** [comment]
Resolved by ioos/catalog-ckan#149.
